.github/workflows/debian.yml

```diff
@@ -0,0 +1,72 @@
+# This action originally modeled off of the Debian build action:
+# https://salsa.debian.org/wouter/ola/-/blob/a38a396f6994b2b1af8efec9e208aee4e67f77aa/.gitlab-ci.yml
+
+name: debian
+on: [push, pull_request]
+jobs:
+  debian-build:
+    name: 'Debian Build ${{ matrix.image_tag }} amd64'
+    runs-on: ubuntu-latest
```

Comment:
Any reason not to use a Debian based container/runner?

Reply:
This is a GH Actions syntax weirdness. GitHub only provides runners natively running Ubuntu, but then see [line 14](https://github.com/OpenLightingProject/ola/pull/1812/files#diff-a57b041fa9cb45a2f493ff99a1b48b1dbd581f0f90dbc905cebb941a314fcf06R14). It runs a Docker container with the official Debian image on top of the Ubuntu runner.

Reply:
Understood, thanks for the explanation!

Reply:
It's containers all the way down!